sqlmesh/core/test/definition.py

```diff
@@ -281,16 +337,17 @@ def __init__(
             models=models,
         )
 
+    def evaluate(self) -> pd.DataFrame:
```

Comment:
We need to disambiguate the name. In a context of a test this reads as it evaluates this test

Reply:
Good point

Reply:
@izeigerman FWIW there's also an `execute` method on the `SqlModel` class. I'm thinking of adding a leading underscore to both of them and renaming this one to `_execute_model` or `_evaluate_model`. What do you think?